heartbeat/hbregistry/hbregistry.go

```diff
@@ -15,32 +15,15 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package scheduler
+package hbregistry
 
-import "sort"
+import "github.com/elastic/beats/libbeat/monitoring"
 
-type timeOrd []*job
+// StatsRegistry contains a singleton instance of the heartbeat stats registry
+var StatsRegistry = monitoring.Default.NewRegistry("heartbeat")
 
-func sortEntries(es []*job) {
-	sort.Sort(timeOrd(es))
-}
+// SchedulerRegistry holds scheduler stats
+var SchedulerRegistry = StatsRegistry.NewRegistry("scheduler")
 
-func (b timeOrd) Len() int {
-	return len(b)
-}
-
-func (b timeOrd) Swap(i, j int) {
-	b[i], b[j] = b[j], b[i]
-}
-
-// Less reports `earliest` time i should sort before j.
-// zero time is not `earliest` time.
-func (b timeOrd) Less(i, j int) bool {
-	if b[i].next.IsZero() {
-		return false
-	}
-	if b[j].next.IsZero() {
-		return true
-	}
-	return b[i].next.Before(b[j].next)
-}
+// StateRegistry contains a singleton instance of the heartbeat state registry
+var StateRegistry = monitoring.GetNamespace("state").GetRegistry().NewRegistry("heartbeat")
```

Comment:
How about naming the `Telemetry` or adding a comment explaining what the two registries are used for?

Reply:
Done!